plugins/modules/win_format.py

```diff
@@ -12,7 +12,7 @@
 options:
   drive_letter:
     description:
-      - Used to specify the drive letter of the volume to be formatted.
+      - Used to specify the drive letter or directory mountpoint of the volume to be formatted.
```

Comment:
You haven't documented the new module option you've added (`filepath`). It sounds like the directory mountpoint is specified there instead.